impl/src/main/java/org/opencord/olt/impl/OltFlowService.java

```diff
@@ -1435,9 +1436,22 @@ public int hashCode() {
         }
     }
 
-    private class InternalFlowListener implements FlowRuleListener {
+    protected class InternalFlowListener implements FlowRuleListener {
         @Override
         public void event(FlowRuleEvent event) {
+
+            if (!appId.equals(event.subject().appId())) {
+                return;
+            }
+
+            if (!oltDeviceService.isLocalLeader(event.subject().deviceId())) {
+                if (log.isTraceEnabled()) {
+                    log.trace("ignoring meter event {} " +
```

Comment:
change the log to "ignoring flow event ...."

Reply:
done